src/UpdatedPackagesCollector.js

```diff
@@ -150,9 +150,9 @@ export default class UpdatedPackagesCollector {
       return file.replace(folder + path.sep, "");
     });
 
-    if (this.flags.ignore) {
+    if (this.flags.publishConfig && this.flags.publishConfig.ignore) {
```

Comment:
Yes, I run [this test case](https://github.com/ant-design/lerna/blob/2099eb8f4fecf36bfe247065b2fc7ff1b620578f/test/UpdatedCommand.js#L228), and get `this.flags` like:

```
  UpdatedCommand
    Circular Dependencies
{ lerna: '__TEST_VERSION__',
  version: '1.0.0',
  publishConfig: { ignore: [ 'ignored-file' ] } }
{ lerna: '__TEST_VERSION__',
  version: '1.0.0',
  publishConfig: { ignore: [ 'ignored-file' ] } }
```